packages/es-components/src/components/navigation/horizontalnav/HorizontalNav.js

```diff
@@ -0,0 +1,75 @@
+/* eslint react/no-unused-prop-types: 0 */
+/* ^^^ This is because defaultSelected needs to be defined to be documented */
+/* The behavior of defaultSelected is implemented in the Uncontrollable library */
+
+import React, { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import styled, { withTheme } from 'styled-components';
+import { noop } from 'lodash';
+import uncontrollable from 'uncontrollable';
+
+import HorizontalNavItem from './HorizontalNavItem';
+
+const NavStyled = styled.nav`
+  background-color: ${props => props.theme.colors.white};
+  ul {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+`;
+
+export const HorizontalNav = props => {
+  const { useAltStyle, children, onItemSelected, selected } = props;
+
+  const onNavClick = (navId = null) => {
+    onItemSelected(navId);
+  };
+
+  return (
+    <NavStyled className="es-horizontalnav">
+      <ul>
+        {Children.toArray(children).map(child => {
+          if (child !== null && child.type === HorizontalNavItem) {
+            const currentSelected = selected;
+            return cloneElement(child, {
+              useAltStyle,
+              highlightedId: currentSelected,
+              onNavClick
+            });
+          }
+          return child;
```

Comment:
Is there a particular reason to alias `selected` into `currentSelected`? I don't see this changing during the `map()` execution to make it unique within the iteration.

Reply:
There isn't, this was just the existing code in SideNav that I copied over. I will update it though.